packages/plugin-core/src/commands/NoteLookupCommand.ts

```diff
@@ -266,10 +268,7 @@ export class NoteLookupCommand
       lc.fuzzThreshold = copts.fuzzThreshold;
     }
 
-    if (!_.isUndefined(this.controller.view)) {
```

Comment:
I think removing this would cause the lookup panel view (most likely fails to render) to pop up during `CreateJournal/ScratchNoteCommand`, which will make the normal lookup view not function as well

Reply:
I moved everything involving `VSCodeUtils.setContext(DendronContext.SHOULD_SHOW_LOOKUP_VIEW, *)` into `LookupPanelView` so that it can also own this part of the panel lifecycle.